ksql-engine/src/main/java/io/confluent/ksql/KsqlContext.java

```diff
@@ -40,38 +43,36 @@
   private final KsqlEngine ksqlEngine;
   private static final String KAFKA_BOOTSTRAP_SERVER_OPTION_DEFAULT = "localhost:9092";
 
-  public static KsqlContext create(KsqlConfig ksqlConfig) {
-    return create(ksqlConfig, null);
+  public static KsqlContext create(final KsqlConfig ksqlConfig) {
+    return create(
+        ksqlConfig,
+        new CachedSchemaRegistryClient(
```

Comment:
Was this just a bug? Also, I think it should be ```new KsqlSchemaRegistryClientFactory(ksqlConfig).create()```

Reply:
Yes, this was here before schema registry integration. We cannot have null schema registry client.
Good point! I changed it to `KsqlSchemaRegistryClientFactory`.